main.py

```diff
@@ -55,16 +73,36 @@ def get_anonymous_commits_stats(repos: list, header: dict) -> dict:
     # see https://docs.github.com/en/free-pro-team@latest/rest/reference/repos#statistics
     stats = {'total': 0}
 
+    if devmode:
+        try:
+            os.mkdir('repo-stats')
+        except FileExistsError:
+            pass
+
     print("\nGetting anonymous commits stats...")
     for i, repo in enumerate(repos):
-        response = requests.get(f"{url_api}/repos/{owner}/{repo}/stats/commit_activity", headers=header)
-        print(f"{i + 1}/{len(repos)} - {repo} - {'OK' if response.status_code == 200 else 'Awaiting new data...'}")
-
-        if response.status_code == 403:
-            raise_rate_limited_exception()
-        elif response.status_code == 200:
-            stats[repo] = sum([weekly['total'] for weekly in response.json()])
-            stats['total'] += stats[repo]
+        if(not (os.path.isfile(f'repo-stats/{repo}.anonymous.json') and devmode)):
+            response = requests.get(f"{url_api}/repos/{owner}/{repo}/stats/commit_activity", headers=header)
+
+            # If in devmode, cache the response in case it does not yet exist
+            if devmode:
+                with open(f'repo-stats/{repo}.anonymous.json', 'w') as f:
```

Comment:
This kind of path concatenations doesn't work on Windows. Since we don't like it, but sometimes we use it, we have to support it. Use `os.path.join(dir, file)` for platform-agnostic code.

Reply:
Fixed with 1b3b965 and tested on Windows 10